metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/ElasticSearchService.java

```diff
@@ -44,6 +46,12 @@ public void configure() {
     indexBuilders.reindexAll();
   }
 
+  @Override
+  public String reindexAsync(String index, @Nullable QueryBuilder filterQuery, BatchWriteOperationsOptions options)
```

Comment:
This also seems out of place...however maybe this class does interact with timeseries indices?